### PR TITLE
TAS-2657/two-layer-verification

### PR DESCRIPTION
**Pull request by marjan-ara**

### CODE REVIEW

File: src/modules/Verifications/components/TypeRadioGroup/useTypeRadioGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { verificationType } from 'src/core/adaptors';
+
+export const useTypeRadioGroup = () => {
+  const cardItems = [
+    {
+      value: 'reusable' as verificationType,
```
Comment (satanmourner) [inline src/modules/Verifications/components/TypeRadioGroup/useTypeRadioGroup.tsx:6]: Don't we have translation for these card items?

File: src/modules/Verifications/components/TypeRadioGroup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { Radio, RadioGroup } from '@mui/material';
+import React from 'react';
+import check from 'src/assets/icons/item-check.svg';
```
Comment (satanmourner) [inline src/modules/Verifications/components/TypeRadioGroup/index.tsx:3]: Don't we have this icon in our icons?

**Review by satanmourner — CHANGES_REQUESTED** [review]

File: src/modules/Verifications/components/TypeRadioGroup/useTypeRadioGroup.tsx (hunk 1 above)
Comment (marjan-ara) [reply src/modules/Verifications/components/TypeRadioGroup/useTypeRadioGroup.tsx:6]: I should Add, thanks for mentioning

File: src/modules/Verifications/components/TypeRadioGroup/index.tsx (hunk 2 above)
Comment (marjan-ara) [reply src/modules/Verifications/components/TypeRadioGroup/index.tsx:3]: it is just used here, we can add it to icon pack later, I will add a FIXME tag

Comment (satanmourner) [reply src/modules/Verifications/components/TypeRadioGroup/index.tsx:3]: we already have it 
you can use "tick" icon inside a square div element

**Review by satanmourner — APPROVED** [review]

**Pull request merged**